server.js

```diff
@@ -2397,13 +2397,14 @@ cache(function(queryParams, match, sendBadge, request) {
 }));
 
 // Coveralls integration.
-camp.route(/^\/coveralls\/([^/]+\/[^/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/coveralls\/(?:(bitbucket)\/)?([^/]+\/[^/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/,
```

Comment:
How about we allow specifying `/github` in the URL here, and updating the examples accordingly as the preferred way to use it? And of course, we can still keep supporting github as a default for backward compatibility, as you're doing.

Reply:
I was afraid that regex ```/^\/coveralls\/(?:(bitbucket|github)\/)...```won't be compatible with previous version for github ```github``` user as shown in example: https://regex101.com/r/3XhFqA/1 (userRepo == ```repo/branch``` instead of ```github/repo```), but ```github``` user currently don't use ```shields.io/coveralls```, so this is not a problem :)